### ISSUE TITLE
Module API examples incorrect for Yarn Plug'n'Play

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/cypress-documentation#5248]
### Subject

Guides > Module API

### Description

https://docs.cypress.io/guides/guides/module-api

For [Yarn Plug'n'Play](https://yarnpkg.com/features/pnp) the command `node e2e-open-tests.js` needs to be executed as:

```bash
yarn node e2e-open-tests.js
```

**Comment by elylucas** [comment]
@MikeMcC399 what is the suggestion here? It doesn't appear we are using yarn in any of the commands here

**Comment by MikeMcC399** [comment]
@elylucas

The command in the documentation `node e2e-open-tests.js` requires that Cypress has been installed. There are six different ways to do this (npm, pnpm, Yarn Classic, then Yarn Modern with its three different `nodeLinker` options: `pnp`, `node-modules` and `pnpm`) and all of them **except** Yarn Modern Plug'n'Play work by just specifying `node`. If you have however installed Cypress with [Yarn Modern in Plug'n'Play mode](https://yarnpkg.com/features/pnp) you need to preface the call with `yarn`. 

I was looking at this in conjunction with the introduction of [Yarn Modern Plug'n'Play support in github-action](https://github.com/cypress-io/github-action#yarn-plugnplay) and the absence of this information threw me to begin with until I got deeper into it. I didn't even know if [Module API](https://docs.cypress.io/guides/guides/module-api) was compatible with Yarn Modern Plug'n'Play from reading the Module API documentation.

package \ command | node | yarn node
--- | --- | ---
npm | y | y
pnpm | y | y
Yarn Classic | y | y
Yarn Modern pnp | x | y
Yarn Modern node-modules | y | y
Yarn Modern pnpm | y | y

The relevant bit of the [Yarn Modern in Plug'n'Play mode](https://yarnpkg.com/features/pnp) documentation says:

> For some remaining edge cases, a small setup may be required:

>
> If you need to run an arbitrary Node script, use [yarn node](https://yarnpkg.com/cli/node) as the interpreter, instead of `node`. This will be enough to register the `.pnp.cjs` file as a runtime dependency.

**Comment by MikeMcC399** [comment]
So the suggestion would be to add a comment to preface the command with `yarn` when Yarn Plug'n'Play is being used, with a reference to Yarn's documentation page https://yarnpkg.com/features/pnp,

**Comment by MikeMcC399** [comment]
I'll make a concrete proposal in a PR for minimal changes to the docs. I think just a note would be sufficient without updating every single example.

**Comment by MikeMcC399** [comment]
> Sorry, not too familiar with yarn pnp.

That makes two of us!

> Is this something that a yarn person would just know they have to do if they are using pnp?

I can't really answer that as I wouldn't consider myself to be a "yarn person". It's probably better not to assume too much prior knowledge.

In any case I've now submitted a suggestion as PR #5254 for you to take a look at. There are currently several issues in Cypress with Yarn Plug'n'Play so I had to go back to `cypress@12.9.0` to demonstrate it. I've tried to keep the text short and simple.
